content/guides/models/track/log/distributed-training.md

```diff
@@ -95,22 +104,21 @@ The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see
 
 ### Track all processes to a single run
 
-
 {{% alert color="secondary"  %}}
 Parameters prefixed by `x_` (such as `x_label`) are in public preview. Create a GitHub issue at [https://github.com/wandb/wandb](https://github.com/wandb/wandb) to provide feedback.
 {{% /alert %}}
 
 {{% alert %}}
-To track multiple processes to a single run, you must have W&B Python SDK version `>=0.19.0`.
+To track multiple processes to a single run, you must have W&B Python SDK version `>=0.19.5`.
```

Comment:
This might be better as "v0.19.5 or newer" or something like that rather than the math shorthand.